boards/arm/mimxrt1050_evk/board.cmake

```diff
@@ -6,10 +6,15 @@
 board_runner_args(jlink "--device=MCIMXRT1052")
 
 if(${CONFIG_BOARD_MIMXRT1050_EVK_QSPI})
+	message("qspi")
     board_runner_args(jlink "--loader=BankAddr=0x60000000&Loader=QSPI")
     board_runner_args(pyocd "--target=mimxrt1050_quadspi")
 else()
+	message("hyper")
     board_runner_args(pyocd "--target=mimxrt1050_hyperflash")
+    board_runner_args(linkserver  "--device=MIMXRT1052xxxxB:EVKB-IMXRT1050")
+    #    board_runner_args(linkserver  "--override=/device/memory/3/flash-driver=MIMXRT1050-EVK_S26KS512S.cfx")
```

Comment:
Should this be commented out? In the same vein, do we need a custom CFX file for the QSPI target of this board?

Reply:
How untidy. LinkServer can find the flash algorithm to use based on the address. This is fixed